examples/00_quick_start/sequential_recsys_amazondataset.ipynb

```diff
@@ -546,135 +554,7 @@
-    "    for batch_data_input in iterator.load_data_from_file(infile, batch_num_ngs=0):\n",
-    "        if batch_data_input:\n",
-    "            feed_dict = {\n",
-    "                model.layer_keeps:np.ones(3, dtype=np.float32),\n",
-    "                model.is_training:False,\n",
-    "                model.items: batch_data_input[iterator.items],\n",
-    "                model.cates: batch_data_input[iterator.cates],\n",
-    "                model.item_history: batch_data_input[iterator.item_history],\n",
-    "                model.item_cate_history: batch_data_input[iterator.item_cate_history],\n",
-    "                model.mask: batch_data_input[iterator.mask],\n",
-    "                model.time_from_first_action: batch_data_input[iterator.time_from_first_action],\n",
-    "                model.time_to_now: batch_data_input[iterator.time_to_now]\n",
-    "            }\n",
-    "            step_pred = sess.run(model.pred, feed_dict=feed_dict)\n",
-    "            preds.extend(np.reshape(step_pred, -1))\n",
-    "                \n",
-    "    with open(outfile, \"w\") as wt:\n",
-    "        for line in preds:\n",
-    "            wt.write('{0}\\n'.format(line))\n",
-    "            "
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "Here is the main pipeline for inferring in an online serving manner. You can compare the 'output_serving.txt' with 'output.txt' to see if the results are consistent.\n",
-    "\n",
-    "The input file format is the same as introduced in Section 1 'Input data format'. In serving stage, since we do not need a groundtrue lable, so for the label column, you can simply place any number like a zero. The iterator will parse the input file and convert into the required format for model's feed_dictionary. "
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 19,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "G = tf.Graph()\n",
-    "with tf.gfile.GFile(\n",
-    "        os.path.join(hparams.MODEL_DIR, \"serving_model.pb\"),\n",
-    "        'rb'\n",
-    ") as f, G.as_default():\n",
-    "    graph_def_optimized = tf.GraphDef()\n",
-    "    graph_def_optimized.ParseFromString(f.read())\n",
-    "    \n",
-    "    ####  uncomment this line if you want to check what conent is included in the graph\n",
-    "    #print('graph_def_optimized = ' + str(graph_def_optimized))\n",
-    "\n",
-    "\n",
-    "with tf.Session(graph=G) as sess:\n",
-    "    tf.import_graph_def(graph_def_optimized)\n",
-    "\n",
-    "    model = LoadFrozedPredModel(sess.graph)\n",
-    "    \n",
-    "    serving_output_file = os.path.join(data_path, r'output_serving.txt')  \n",
-    "    iterator = input_creator(hparams, tf.Graph())\n",
-    "    infer_as_serving(model, test_file, serving_output_file, hparams, iterator, sess)\n",
-    "    "
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "## Reference\n",
```

Comment:
Since this part is removed, I think we should change the section name from "Online Serving" to "Loading Trained Models"

Reply:
Done.